src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -516,14 +516,7 @@ public function handleException(\Throwable $exception): void
             }
         }
 
-        if (!$exception instanceof OutOfMemoryError) {
-            foreach ($this->getErrorEnhancers() as $errorEnhancer) {
-                if ($e = $errorEnhancer->enhance($exception)) {
-                    $exception = $e;
-                    break;
-                }
-            }
-        }
+        $exception = self::enhanceError($exception);
```

Comment:
this should be `$this` instead of `self`, right?

so if a sub-class is overriding the `enhanceError()` method, it will be called for sure.

Reply:
of course, I forgot to revert this change! Thanks (fixed)